client/src/core/client/admin/components/MediaContainer/BlueskyMedia.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from "react";
+
+import Frame from "coral-framework/components/Frame";
+
+import styles from "./Media.css";
+
+interface Props {
+  id: string;
+  url: string;
+  siteID: string;
+}
+
+const BlueskyMedia: FunctionComponent<Props> = ({ id, url, siteID }) => {
+  const component = encodeURIComponent(url);
+  return (
+    <div className={styles.embed}>
+      <Frame
+        id={id}
+        src={`/api/oembed?type=twitter&url=${component}&siteID=${siteID}`}
```

Comment:
I think `type` here should equal `bsky`?

Reply:
fixed